src/components/NewWorkspaceModal.ts

```diff
@@ -319,11 +319,19 @@ const NewWorkspaceModal = withDisplayName(
     };
 
     const isBillingProjectApplicable = (project: BillingProject): boolean => {
-      // As of AJ-1164, if requireEnhancedBucketLogging is true, then azure billing projects are ineligible.
-      // This coupling of enhanced bucket logging and billing project may change in the future.
-      if (!!workflowImport || !!requireEnhancedBucketLogging) {
+      // This is used when importing data to enforce a specific cloud.
+      if (cloudPlatform && project.cloudPlatform !== cloudPlatform) {
+        return false;
+      }
+      if (workflowImport) {
         return !isAzureBillingProject(project);
       }
+      // If creating a new workspace and enhanced bucket logging is required, allow all GCP projects
```

Comment:
I think the text in the comment is really helpful but its placement is a bit confusing because it's describing more than just the conditional that immediately follows it, and it isn't actually describing _all_ the interceding logic up through line 344 because there's a little conditional in there that only applies to cloned workspaces.

It might be helpful to break the comment apart so that each clarifying statement is closer to the `if` being described.

Reply:
`!clonedWorkspace` === "new workspace", so really this comment is only about lines 332-334. I can update the comment to state "If we aren't cloning a workspace"